src/modules/pece/pece_security/pece_security.features.user_permission.inc

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * @file
+ * pece_security.features.user_permission.inc
+ */
+
+/**
+ * Implements hook_user_default_permissions().
+ */
+function pece_security_user_default_permissions() {
```

Comment:
Those permissions should not be exported using Permissions from Features module so they can be overridden. See above.